app/uk/gov/hmrc/agentsubscriptionfrontend/controllers/package.scala

```diff
@@ -40,11 +42,13 @@ package object controllers {
       }
     }
 
-    private val nonEmptyTelephoneNumber: Constraint[String] = Constraint[String] { fieldValue: String =>
```

Comment:
Shouldn't we be consistent with the other variables? If I change this shouldn't we change all of them?